packages/ui/src/components/va-option-list/VaOptionList.vue

```diff
@@ -2,61 +2,61 @@
   <VaMessageListWrapper
     :error="computedError"
     :error-messages="computedErrorMessages"
-    :error-count="$props.errorCount"
+    :error-count="errorCount"
```

Comment:
Discussed with @RVitaly1978,
there is no much need to remove $props, as they serve as indication you're using props directly, and not computed of sorts.